### PR TITLE
Update DEFINITION.md

### PR DESCRIPTION
**Pull request by TaniaFedirko**
Signed-off-by: Tania Fedirko <66258087+TaniaFedirko@users.noreply.github.com>
Add Ukrainian Definition

### CODE REVIEW

**Comment by TaniaFedirko** [comment]
Add Ukrainian Definition

**Comment by craigbox** [comment]
Spot check with Google Translate LGTM.  
Adding a couple of potential reviewers: @idvoretskyi @kivashch to get this moving.

File: DEFINITION.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ ecosystem of open source, vendor-neutral projects. We democratize state-of-the-a
 innovations accessible for everyone.
 
 
+## Ukrainian 🇺🇦
+
+Нативні хмарні технології дозволяють організаціям створювати та запускати масштабовані додатки в сучасних динамічних середовищах, таких як публічні, приватні та гібридні хмари. Контейнери, сервісні сітки, мікросервіси, незмінна інфраструктура і декларативні API є прикладами такого підходу.
+
+Ці методи дозволяють створювати слабко пов'язані системи, які є стійкими, керованими і спостережуваними. У поєднанні з надійною автоматизацією вони дозволяють інженерам часто і передбачувано вносити значні зміни з мінімальними зусиллями.
+
+Фундація хмарних нативних обчислень прагне сприяти впровадженню цієї парадигми шляхом розвитку і підтримки екосистеми проектів з відкритим вихідним кодом, незалежних від постачальників. Ми демократизуємо найсучасніші моделі, щоб зробити ці інновації доступними для кожного.
```
Comment (idvoretskyi) [inline DEFINITION.md:27]: s/постачальників/вендорів/

File: DEFINITION.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ ecosystem of open source, vendor-neutral projects. We democratize state-of-the-a
 innovations accessible for everyone.
 
 
+## Ukrainian 🇺🇦
+
+Нативні хмарні технології дозволяють організаціям створювати та запускати масштабовані додатки в сучасних динамічних середовищах, таких як публічні, приватні та гібридні хмари. Контейнери, сервісні сітки, мікросервіси, незмінна інфраструктура і декларативні API є прикладами такого підходу.
+
+Ці методи дозволяють створювати слабко пов'язані системи, які є стійкими, керованими і спостережуваними. У поєднанні з надійною автоматизацією вони дозволяють інженерам часто і передбачувано вносити значні зміни з мінімальними зусиллями.
```
Comment (idvoretskyi) [inline DEFINITION.md:25]: > слабко пов'язані системи

Please rephrase.

File: DEFINITION.md [hunk 3]
```diff
@@ -18,6 +18,14 @@ ecosystem of open source, vendor-neutral projects. We democratize state-of-the-a
 innovations accessible for everyone.
 
 
+## Ukrainian 🇺🇦
+
+Нативні хмарні технології дозволяють організаціям створювати та запускати масштабовані додатки в сучасних динамічних середовищах, таких як публічні, приватні та гібридні хмари. Контейнери, сервісні сітки, мікросервіси, незмінна інфраструктура і декларативні API є прикладами такого підходу.
```
Comment (idvoretskyi) [inline DEFINITION.md:23]: > сервісні сітки

I believe, Service Meshes are meant here, I'd look for a better translation of the definition.

**Review by idvoretskyi — CHANGES_REQUESTED** [review]

**Comment by TheFoxAtWork** [comment]
I would love for this to be merged in but there are some conflicts and outstanding comments that need resolved, @TaniaFedirko our apologies on the delay, would you update the PR?

**Comment by TheFoxAtWork** [comment]
@TaniaFedirko & @idvoretskyi  would you take a look?

**Comment by idvoretskyi** [comment]
Ping @TaniaFedirko :)

**Comment by TheFoxAtWork** [comment]
Closing this PR - internationalism of the definition will need refreshed with the New Definition to be merged. 
CC @jeefy

**Pull request closed without merging**